spec/workers/calculate_project_activity_worker_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'spec_helper'
+
+describe CalculateProjectActivityWorker do
+  let(:worker) { described_class.new }
+  let(:workflow) { create(:workflow) }
+  let(:project) { workflow.project }
+  let(:user) { project.user }
+
+  let(:classifications) do
+    create_list(:classification, 2, project: project, workflow: workflow)
+    c = create(:classification, project: project, workflow: workflow)
+    c.update_column(:created_at, 25.hours.ago)
+  end
+
+  describe '#perform' do
+
```

Comment:
What's up with the blank lines after a `describe` or `context`? Do we have a style guide about this?